src/com/owncloud/android/ui/activity/PatternLockActivity.java

```diff
@@ -0,0 +1,270 @@
+package com.owncloud.android.ui.activity;
```

Comment:
Include the same as https://github.com/owncloud/android/pull/2083/files#r165983321